src/components/Tool/Appeal/InitialPage/AddAppealForm/AddAppealForm.tsx

```diff
@@ -289,35 +264,22 @@ const AddAppealForm: React.FC<AddAppealFormProps> = ({
   const { t } = useTranslation();
   const { push } = useRouter();
   const { enqueueSnackbar } = useSnackbar();
+  const { contactStatuses } = useContactPartnershipStatuses();
+  const statusOptions = Object.entries(contactStatuses).map(
+    ([statusId, status]) => {
+      return {
+        name: status.translated,
+        value: statusId,
+      };
+    },
+  );
   const { data: contactFilterTags, loading: loadingTags } = useContactTagsQuery(
     {
       variables: { accountListId },
     },
   );
-  const { data: contactFilterGroups, loading: loadingStatuses } =
-    useContactFiltersQuery({
```

Comment:
Since you're removing this graphQL, You can remove it also from he graphQL page since I believe no other pages use it.